httplogs/utility.go

```diff
@@ -0,0 +1,31 @@
+package httplogs
+
+import (
+	"net"
+	"net/http"
+)
+
+func getValueFromConsulByKey(key string) string {
+	return string(_gLogConfig.consulAgent.GetValue(key))
```

Comment:
check what happens if the value returned is nil. string(nil).. not sure it will work.

Reply:
there is no issue with that